python/vineyard/deploy/vineyard_k8s.yaml.tmpl

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: vineyard
+  namespace: vineyard
+  labels:
+    app: vineyard
+spec:
+  selector:
+    matchLabels:
+      name: vineyard
+  template:
+    metadata:
+      labels:
+        name: vineyard
+    spec:
+      tolerations:
+      # this toleration is to have the daemonset runnable on master nodes
+      # remove it if your masters can't run pods
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: vineyard
+        image: quay.io/libvineyard/vineyardd:latest
+        resources:
+          limits:
+            memory: {{ MemorySize }}
+          requests:
+            cpu: 100m
+            memory: {{ MemorySize }}
```

Comment:
The args "size" is not the "memory limit" in resources, it is the size of shared memory that vineyardd can use, i.e., it is the `--size` command line argument for `vineyardd`.